pom.xml

```diff
@@ -4,14 +4,18 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.532.3</version>
+    <version>2.31-SNAPSHOT</version>

```

Comment:
True, I wanted to check compatibility going forward. That said, would you be all right with depending on the latest stable version (2.32)?